e2e/page_objects/programCatalog.js

```diff
@@ -5,24 +5,36 @@ class ProgramCatalog {
     constructor() {
         this.logo = element(by.css('.header__navbar--logo'));
         this.searchBar = element(by.css('.adjust-search input'));
-        this.resultCounter = element(by.css('.result-count'));        
+        this.resultCounter = element(by.css('.result-count'));
+        this.bannerCloseButton = element(by.css('.close-btn'));
 
         this.cards = element.all(by.css('.card__inner'));
         this.selectedFiltersField = element.all(by.css('.filters'));
 
         this.filterBox = text => element(by.cssContainingText('.multiselect-dropdown', text));
-        this.dropdownList = text => this.filterBox(text).element(by.css('.dropdown-list'));
         this.filterOption = text => element(by.cssContainingText('.multiselect-item-checkbox', text));
+        this.findCard = text => element(by.cssContainingText('ir-catlog-card div.card-wrapper', text));
```

Comment:
Please, don't use 'find' in the selectors' name. It is enough to use card or cardTitle.